### PR TITLE
remove obsolete files

### PR DESCRIPTION
**Pull request by dtrifiro**
In 80a5669...5de0b4c tiny models were added explicitly for testing, so we should not need docker-compose anymore

### CODE REVIEW

File: Makefile
Comment (z103cb) [inline Makefile:1]: can we leave the make file in the repo an just adjust the targets? I find it handy to have the Makefile to run the tests / formatting.

**Review by z103cb — COMMENTED** [review]
I am not too sure that we want to nuke the Makefile wholesale, but  I will abide by your decision. @vaibhavjainwiz what is your take ?

File: Makefile
Comment (dtrifiro) [reply Makefile:1]: That's what `nox` is for:

```bash
nox -s <session>
```

- `tests` session runs tests
- `pre-commit` runs `formatting` and linting
etc

three python versions are used by default (`python_versions = ["3.11", "3.10", "3.9"]`), but you can specify a single version using `--python`.

We can add more sessions if you feel like we're missing something.



https://github.com/vaibhavjainwiz/caikit-nlp-client/blob/main/noxfile.py

Comment (z103cb) [reply Makefile:1]: Ok, makes sense.

**Comment by z103cb** [comment]
LGTM.

**Pull request merged**